### PR TITLE
Problem: Minor inconsistency in macro syntax. Solution: Code fixed.

### PR DESCRIPTION
**Pull request by bjovke**

### CODE REVIEW

File: src/proxy.cpp [hunk 1]
```diff
@@ -69,7 +69,7 @@
         delete poller_both_blocked;\
         delete poller_frontend_only;\
         delete poller_backend_only;\
-    } while (0)
+    } while (true)
```
Comment (bluca) [inline src/proxy.cpp]: surely you meant while (false) ?

Comment (bjovke) [inline src/proxy.cpp]: @bluca Actually yes.....

Comment (bjovke) [inline src/proxy.cpp]: I can't believe I did this...

**Comment by bjovke** [comment]
@bluca Sorry. Now it should be OK. Two brains are better than one.

**Comment by bluca** [comment]
no worries

**Pull request merged**